lib/uring/uring.mli

```diff
@@ -278,36 +432,66 @@ module Statx : sig
   end
 
   module Mask : sig
+    (** The mask flags are used to tell the kernel which fields the {!statx} invocation
+        is interested in. You may wish to use {! Mask.check} on the returned {!Statx.t} to
+        verify the field has actually been filled in with a sensible value first.
+        In general, the kernel never refused values specified in the mask, but may choose
+        to not set the mask in the returned buffer from {!statx}. *)
```

Comment:
Where is this from? statx(2) says:
> It should be noted that the kernel may return fields that weren't requested and may fail to return fields that were  requested

Reply:
That quote is compatible with the ocamldoc above. From https://man7.org/linux/man-pages/man2/statx.2.html

       Note that, in general, the kernel does not reject values in mask
       other than the above.  (For an exception, see EINVAL in errors.)
       Instead, it simply informs the caller which values are supported
       by this kernel and filesystem via the statx.stx_mask field.
       Therefore, do not simply set mask to UINT_MAX (all bits set), as
       one or more bits may, in the future, be used to specify an
       extension to the buffer.

Reply:
OK, I guess the meaning of "reject/refuse" here was confusing me.

I read the man-page as saying that you won't get `EINVAL` for specifying an unknown mask bit; instead it will just ignore the unknown bits for forwards-compatibility.

But `the kernel never refuse[s] values specified in the mask` could also be interpreted as saying that the kernel will always return the values you ask for.